integration/v3_grpc_inflight_test.go

```diff
@@ -81,8 +82,9 @@ func TestV3KVInflightRangeRequests(t *testing.T) {
 			defer wg.Done()
 			_, err := kvc.Range(ctx, &pb.RangeRequest{Key: []byte("foo"), Serializable: true}, grpc.FailFast(false))
 			if err != nil {
-				if err != nil && rpctypes.ErrorDesc(err) != context.Canceled.Error() {
-					t.Fatalf("inflight request should be canceld with %v, got %v", context.Canceled, err)
+				errDesc := rpctypes.ErrorDesc(err)
+				if err != nil && !(errDesc == context.Canceled.Error() || errDesc == transport.ErrConnClosing.Desc) {
```

Comment:
@gyuho Is this acceptable? I've tried to reason through it and it *seems* reasonable to me that a client would get 'connection closing' errors here. But this is different than the previous behavior.

Also, we could alternatively (or as an additional test) switch the order of `clus.Members[0].Stop(t)` and `cancel()
` (below) in this test and it should then always report 'context canceled' errors.

Reply:
@jpbetz Just confirmed that this is correct with new gRPC. Previously, it only called `getTransport`, so you only get errors from client-provided context. Now, `transport.ErrConnClosing` is returned from `newClientStream` before canceling the client-provided context, which explains better about closed server. So much have happened in gRPC since v1.7, but I assume this change is from https://github.com/grpc/grpc-go/commit/d09ec435457694f2e314397fe5bc18a1d959f303.

Reply:
Thanks for checking up on this!